pkg/cli/flags.go

```diff
@@ -29,6 +29,7 @@ const (
 	ArtifactFlagName              = "--artifact"
 	BootstrapServersFlagName      = "--bootstrap-servers"
 	CacheSizeFlagName             = "--cache-size"
+	ContainerConcurrencyFlagName  = "--container-concurrency"
```

Comment:
Do we want to call this something like `concurrency-limit` or just `concurrency`?